dashboard/src/components/CreateActionModal.js

```diff
@@ -191,15 +190,17 @@ const CreateActionModal = ({ person = null, persons = null, isMulti = false, com
                     <FormGroup>
                       <Label htmlFor="dueAt">À faire le</Label>
                       <div>
-                        <DatePicker
-                          id="dueAt"
+                        <input
+                          className="form-control placeholder:tw-opacity-0"
+                          type={values.withTime ? 'datetime-local' : 'date'}
```

Comment:
Je pense que c'est l'occasion de mettre des min max sur les dates

```suggestion
                          type={values.withTime ? 'datetime-local' : 'date'}
                          min="1901-01-01"
                          max="2200-01-01"
```